src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -121,8 +121,7 @@
          Note that queues will never have ',', '+' or ';' in their name, and this greatly simplifies providing a queue list in *nix.
     -->
     <PropertyGroup>
-      <ProcessedTargetQueues>$([System.String]::Copy('$(TargetQueues)').Replace(',',';'))</ProcessedTargetQueues>
-      <ProcessedTargetQueues>$([System.String]::Copy('$(TargetQueues)').Replace('+',';'))</ProcessedTargetQueues>	  
+      <ProcessedTargetQueues>$([System.String]::Copy('$(TargetQueues)').Replace(',',';').Replace('+',';'))</ProcessedTargetQueues>  
```

Comment:
Just because they can doesn't mean they should. We should try and consistently use one delimiter if possible.